src/foam/nanos/logger/LogMessageDAO.js

```diff
@@ -40,6 +40,7 @@ foam.CLASS({
       javaCode: `
       LogMessage lm = (LogMessage) obj;
       lm.setHostname(getHostname());
+      lm.setLastModified(new java.util.Date());
```

Comment:
Do we never load from the DAO on restart?

Reply:
Nope, the ID is also storageTransient. 
It's backed by WriteOnly and FixedSize DAOs